service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -113,8 +116,8 @@ private boolean checkSchemaIdSet() {
     schemaIdSetMatch = existSchemas.equals(localSchemas);
 
     if (!schemaIdSetMatch) {
-      LOGGER.error(
-          "SchemaIds is different between local and service center. Please change microservice version. "
+      LOGGER.warn(
+          "SchemaIds is different between local and service center. "
```

Comment:
old suggestion is change version.
what's the new suggestion?

Reply:
done, still use the old suggestion